test/test_prototype_transforms.py

```diff
@@ -1885,3 +1886,67 @@ def test__transform(self, inpt):
         assert type(output) is type(inpt)
         assert output.shape[-4] == num_samples
         assert output.dtype == inpt.dtype
+
+
+# TODO: remove this test in 0.17 when the default of antialias changes to True
+def test_antialias_warning():
```

Comment:
I'm mixing the functional and class tests here... Hope that's fine (I found it to be a awkward to split them as done in the V1 tests)

Reply:
Fine by me. Test is self-contained enough that there shouldn't be any confusion about it later.